### ISSUE TITLE
Find and lock in locations for worldwide meetup

### ISSUE THREAD

**Issue by erickt** [rust-community/team#164]
If possible, we should find locations around the world that can host local events. At a minimum, these spaces will need to be able to project the live stream of the event. It would be even better if the spots could support a local speaker to present as well.

**Comment by Byron** [comment]
Are people supposed to leave a note here with information about the location?

**Comment by erickt** [comment]
@Byron Yes, that'd be great if you have a location in mind. I'm planning on organizing the San Francisco Mozilla office for this.

**Comment by Byron** [comment]
@erickt I would actually work towards making many ThoughtWorks Germany offices available for hosting this event. Even though I will only be present in the one in Hamburg, @nhellwig might be able to provide the event space in Cologne. I know @killercup and friends are hosting great meetups, and I would just put the TW-Cologne space out there as an option in case they need it.

Depending on the time of the worldwide meetup, ThoughtWorks Berlin might already have their brand new office ready to host this even too :).

**Comment by badboy** [comment]
@Byron Thanks for the offer! We will talk to @nhellwig and see what the best options are (either TW or the c4)
